ui/raidboss/data/07-dt/raid/r5s.txt

```diff
@@ -1,9 +1,364 @@
-### Arcadion (R5S): AAC Cruiserweight M1 Savage
+### AAC CRUISERWEIGHT M1 (SAVAGE)
+# ZoneId: 1257
+
+# -ii 93C1 93C2 93C3 93C8 93C9 A327 A70A A70B A70C A70D A77A
+
+# The X-snap twist syncs are ugly, sorry
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" ActorControl { command: "4000000F" } window 0,100000 jump 0
 
+# Intro
 0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+15.3 "Deep Cut" Ability { id: "A722", source: "Dancing Green" }
+20.9 "--middle--" Ability { id: "A6C5", source: "Dancing Green" }
+27.0 "Flip to A-side/Flip to B-side" Ability { id: ["A780", "A781"], source: "Dancing Green" }
```

Comment:
I think we could add a timeline `replaceText` here to shorten to `Flip to A/B-side` for display.